mcux/mcux-sdk/drivers/common/fsl_common.h

```diff
@@ -222,6 +222,9 @@ enum
 /*! @brief Type used for all status and error return values. */
```

Comment:
Ideally there should be no difference between SDK repo and this repo modulo some specific files that are only used in hal_nxp.

Not sure why this can't be merged to SDK repo? It shouldn't bother anyone but I think you know better so I don't have anything against this change.

Reply:
I will check with the SDK team if this can be merged.